src/models/bridge-data-filter.model.ts

```diff
@@ -7,8 +8,9 @@ export class BridgeDataFilterModel {
   }
 
   isMethodCall(callData: string) {
-    return callData.startsWith(this.abiEncodedSignature) ||
-      callData.startsWith('0x' + this.abiEncodedSignature);
+    const data = ensure0x(callData);
+    const methodSignature = data.slice(0, 10);
+    return methodSignature === this.abiEncodedSignature;
```

Comment:
Please, remind me something. The encoded signature comes with 0x?
Seeing the old code it makes me think it does not. As we were comparing `callData` with `'0x' + this.abiEncodedSignature` and if `this.abiEncodedSignature` has `0x` we would have been duplicating the `0x`...

Can you verify this please?

Reply:
The encoded signature comes with 0x. The old code was ensuring the data started with the 0x, in case it did not, it prepended it. Which is also what I'm doing here, just in case.